requirements.txt

```diff
@@ -12,3 +12,4 @@ six
 setuptools>=18.0
 pyyaml
 xarray
+statsmodels
```

Comment:
@mcflugen: Are you implying that we should include another method that emulates the empirical_distribution in our component? Or should leave stats_model as a requirement?